src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,71 @@
-import React from 'react';
+import React, { FC, useState } from 'react';
+import cn from 'classnames';
+import { Movie } from '../../types/Movie';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  handlerAdd: (newMovie: Movie) => void;
+}
+
+const baseImgUrl = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+export const FindMovie: FC<Props> = ({
+  handlerAdd,
+}) => {
+  const [hasError, setHasError] = useState(false);
+  const [selectedMovie, setSelectedMovie] = useState<Movie | null>(null);
+  const [searchQuery, setSearchQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleQueryChange = (value: string) => {
+    setSearchQuery(value);
+
+    if (value) {
+      setHasError(false);
+    }
+  };
+
+  const handleAddMovieToTheList = () => {
+    if (selectedMovie) {
+      handlerAdd(selectedMovie);
+      setSelectedMovie(null);
+      setSearchQuery('');
+    }
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
use useCallback here
Also, would be better to use `async` function & `await` your response